src/Sarif.Viewer.VisualStudio.Core/ICodeAnalysisResultManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal interface ICodeAnalysisResultManager
+    {
+        List<string> TryResolveFilePaths(RunDataCache dataCache, string workingDirectory, string logFilePath, List<string> uriBaseIds, List<string> relativePaths);
```

Comment:
>TryResolveFilePaths [](http://example.com/codeflow?start=21&length=19)

Since this is a Try method, it should return bool and have an out param for the List<>

Reply:
renamed to ResolveFilePaths